cadnano/views/gridview/tools/abstractgridtool.py

```diff
@@ -250,14 +250,14 @@ def setActive(self, will_be_active, old_tool=None):
         if self._active and not will_be_active:
             self.deactivate()
         self._active = will_be_active
-        self.sgv = self.manager.window.grid_graphics_view
+        self.slice_graphics_view = self.manager.window.grid_graphics_view
         if hasattr(self, 'getCustomContextMenu'):
             # print("connecting ccm")
             try:    # Hack to prevent multiple connections
-                self.sgv.customContextMenuRequested.disconnect()
-            except Exception:
+                self.slice_graphics_view.customContextMenuRequested.disconnect()
+            except AttributeError:
```

Comment:
Updating `except Exception` blocks to catch specific exceptions rather than the broad, base `Exception`